docs/src/guides/general/default-branch.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Renaming a project's parent environment"
+sidebarTitle: "Rename the parent environment"
+description: |
+    By default, Platform.sh considers `master` to be the parent branch for all projects, making Master the parent and production environment. With this guide, you can change a `default_branch` property on your projects to match an externally integrated repository on GitHub, GitLab, or Bitbucket.
+---
+
+{{< description >}}
+
+Soon you will be able to select your default branch during the project creation steps, but until that time this guide will show you how to manually make the switch yourself. You can complete some of these steps through the management console, but since all can be completed with the CLI those commands alone are listed. Be sure to [install the CLI](http://localhost:1313/development/cli.html#installation) if you have not already done so. It's assumed you are changing the default branch of your project on Platform.sh from `master` to `main`. If using another name for the default branch, update the commands accordingly. 
+
+## New projects
+
+The steps below assume:
+
+- You have an external repository on GitHub, GitLab, or BitBucket you are trying to integrate with Platform.sh.
+- That repository's default branch is not `master`.
+
+### 1. Create a new project
+
+First, create an empty project with the Platform.sh CLI command `platform create`:
+
+```bash
+$ platform create --title='Main Project' --region=us-3.platform.sh --plan=development --environments=3 --storage=5 --no-set-remote
+```
+
+This will create a new project with Master as the parent environment by default. Modify the flags to fit your use case, otherwise exclude and the CLI will ask you to set them individually during creation. Copy the `Project ID` provided when the command completes, and substitute the value in the steps below. You can also visit the provided management console URL for the project (i.e. `https://console.platform.sh/<USER>/<Project ID>`) to verify the steps as you go along.
```

Comment:
```suggestion
This will create a new project with Master as the parent environment by default. Modify the flags to fit your use case, or skip them and the CLI will ask you to set them individually during creation. Copy the `Project ID` provided when the command completes, and substitute the value in the steps below. You can also visit the provided management console URL for the project (i.e. `https://console.platform.sh/<USER>/<Project ID>`) to verify the steps as you go along.
```